DigitalExcellence/dex-frontend
# Feature/313 update alt tags

## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Run the website
2.  Check if images have descriptive alt tags


## Link to issue

Closes: #313
